tests/std/tests/GH_001858_iostream_exception/test.cpp

```diff
@@ -0,0 +1,102 @@
+// Copyright (c) Microsoft Corporation.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+#include <cassert>
+#include <istream>
+#include <ostream>
+
+using namespace std;
+
+template <class CharT>
+class throwing_buffer : public basic_streambuf<CharT> {
+public:
+    streampos seekoff(streamoff, ios::seekdir, ios_base::openmode = ios_base::in | ios_base::out) override {

```

Comment:
Style nitpick, this uses both `ios::` (which is a typedef for `basic_ios<char>`) and `ios_base::` (the base class that defines these types etc.). Either is fine (I personally prefer `ios_base::`) but I recommend being consistent.